### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thirulak**

### CODE REVIEW

**Comment by justinclift** [comment]
Thanks @thirulak.

Looking over this, it has some misspellings.  Are you ok with me fixing those before I merge this? :smile:

As a note - don't be too worried about the misspellings.  We all have days like that, and you're obviously still learning English (a good thing) at the moment. :wink:

**Comment by thirulak** [comment]
Thanks @justinclift 
I will try to fix the misspellings

File: README.md [hunk 1]
```diff
@@ -446,3 +446,8 @@ Open source means that someone from Australia can make a Pull Request to help me
 
 Open source to me is all about the community and the discussions which are a means for collective learning. It is a better learning medium than even a classroom because you can talk and learn from experts in their own fields.
 Open source projects are a living, breathing entity constantly being shaped and molded by each contributor as they addd a little bit of themselves into the project.
+
+### [Thirumalai devi ramya](https://github.com/thirulak/thirulak.github.io)
```
Comment (justinclift) [inline README.md]: Should that link instead be: https://thirulak.github.io ? :smile:

Comment (thirulak) [reply README.md]: Yea thanks for giving a note on where i made a mistake

**Comment by justinclift** [comment]
Excellent.  It looks good now, so I'll merge this.

Thanks for adding yourself here.  Many of the people who have added themselves here, have gone onwards to accomplish significant/great things.  Hopefully you do as well. :smile:

**Comment by nickdesaulniers** [comment]
Thanks @thirulak !  In regards to learning from reading open source:

What did I find the other day reading the source to [abseil](https://github.com/abseil/abseil-cpp)...hmm I think they had a way to detect if a parameter was a string literal vs a variable identifier...ugh now I can't find it.  Many gems in there.

**Pull request merged**